### PR TITLE
Cassey-community: fix img HTML issues in magikarp on no js page & parsing errors on a project page

### PR DESCRIPTION
**Pull request by clottman**
## Links
* https://cassey-community.glitch.me (see how-to-test section)
* https://glitch.manuscript.com/f/cases/3328751/A11y-issues-on-magikarp-on-you-need-JS-fallback-page
* https://glitch.manuscript.com/f/cases/3328753/ 

## Changes:
* Width is automatically in pixels, so valid values for the 'width' attribute are numbers only. 
* Add alt=''

## How To Test:
For magikarp: 
*  Go to [cassey-community in the W3 HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fcassey-community.glitch.me#l243c984) and you shouldn't see any warnings, compared to [glitch.com in the W3 HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fglitch.com#l243c984) where you do.

For other changes: 
image
- [HTML Checker bookmarklets can be found at this link under extras](https://validator.w3.org/nu/about.html)
- Make sure no visual regressions happened due to these changes

Caveats: 
 - figuring out how to remove the improper nesting for tooltips proved challenging; going to do that in a later PR. Also, the allowvr attribute isn't part of the spec yet but we need it for displaying embeds with VR content. You should see these two errors (and only these two) when you run the html checker on cassey-community.glitch.me/~one-of-your-projects. 
image

### CODE REVIEW

**Comment by keithk** [comment]
oops sorry about that -- moving things around in zenhub!

File: src/components/project/embed.js [hunk 1]
```diff
@@ -31,13 +31,11 @@ const Embed = ({ domain }) => (
       // Embed iframe for app
       <iframe
         className={styles.embedIframe}
-        title="embed"
         src={`${APP_URL}/embed/#!/embed/${domain}?path=README.md&previewSize=100`}
-        alt={`${domain} on Glitch`}
+        title={`${domain} on Glitch`}
```
[the remark this answers is not in the document]
Comment (clottman) [reply src/components/project/embed.js]: Iframes don't have `alt` attributes, per the spec. So it's unlikely assistive technologies will pick up the `alt` value as part of the accessible name of the iframe, and the `alt` will instead be ignored altogether.

File: src/components/project/project-embed.js [hunk 2]
```diff
@@ -28,9 +28,9 @@ const ProjectEmbed = ({ project, top, isAuthorized, currentUser, addProjectToCol
   const BottomRight = () => (
     <>
       {currentUser.login && (
-        <span className={styles.addToCollectionWrap}>
+        <div className={styles.addToCollectionWrap}>
```
[the remark this answers is not in the document]
Comment (clottman) [reply src/components/project/project-embed.js]: That's a great idea! I hadn't really thought about it before this either and also didn't know the difference. I can do some more research and bring it to a future check-in as the tip of the week!

**Comment by clottman** [comment]
@sarahzinger 

> would you like us to do this sort of check frequently?
Ideally, yes! I'm not sure how possible it is to automate though - the current testing workflow is quite manual and I'd love to have this be in an automated test on each pull request instead.

> do we not have to worry about the errors you get when you do "check serialialized dom of current page" and we just care about the wcag 2.0 parsing compliance?
Correct. We care about things that will prevent assistive technologies from being able to understand what's on the page, which primarily means we care about missing start or end tags, improper nesting, and duplicate attributes/re-used IDs. 

> I see in your caveats you mention 2 errors, i see 4? Still an improvement tho so let's ship! (also it's super possible I'm confused at what I'm looking at this is my first time using these tools)
Interesting! Are you logged in? What page are you on, and can you send a screenshot or upload a snippet of HTML around where you're seeing anything with class="callouts" or a <figure> as a child of span? I don't seem to have that on ~soulpatch.

File: src/components/project/embed.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarahzinger) [reply src/components/project/embed.js]: ahhhh i didn't know that thanks!

File: src/components/project/project-embed.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (sarahzinger) [reply src/components/project/project-embed.js]: rad thanks :)

File: src/components/project/embed.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (clottman) [reply src/components/project/embed.js]: Nor did I! Learned while digging into the provided extra info from this tool.

**Comment by sarahzinger** [comment]
@clottman awesome, yeah now I only see 2 again? not sure what I was doing maybe I was on the homepage or something. Anyway back to 2 looks good!

**Pull request merged**